kernel/src/drivers/mod.rs

```diff
@@ -0,0 +1,10 @@
+use log::info;
```

Comment:
This module seems kind of all over the place. I think for now it might be better to keep these submodules at crate root or to find a different way of categorizing these.